ocaml/xapi/message_forwarding.ml

```diff
@@ -1334,7 +1378,8 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
 							end
 						else
 							(* if we're nt suspended then just forward to host that has vm running on it: *)
-							forward_vm_op ~vm in
+							do_op_on ~host:host
```

Comment:
I think we need `do_op_on`, After cancellation of all tasks we can get the VM state as `Halted which will fail in `forward_vm_op`.
```
let forward_vm_op ~local_fn ~__context ~vm op =
	Xapi_vm_lifecycle.assert_power_state_in ~__context ~self:vm ~allowed:[`Running; `Paused];
	do_op_on ~local_fn ~__context ~host:(Db.VM.get_resident_on ~__context ~self:vm) op
```